proto/abis/massa/abi/v1/abi.proto

```diff
@@ -691,24 +593,24 @@ message VerifyBlsMultiSigResult {
   bool is_verified = 1;
 }
 
-// Native hash request
-message NativeHashRequest {
+// Blake3 hash request
+message Blake3HashRequest {
   // Data to hash
   bytes data = 1;
```

Comment:
As I understand, we no longuer have the version byte for the hash. Does this mean if we change the hash in Massa we will need to have a breaking ABI change?
Follow-up question, would we ever want to query XOF hashes in the ABI?

Reply:
* that's why it was renamed "blake3 hash" => this unambiguously defines what it is. There is no concept of native hash
* good point, but this might be a followup ABI (not for now)